commit 13a8894fe2faa45b04033d7122a8fe7939ce6aa2
Author: Gavin Smith <gavinsmith0123@gmail.com>
Date:   Thu Sep 16 19:06:28 2021 +0100

    * tp/Texinfo/XS/parsetexi/end_line.c (parse_line_command_args)
    <@headings>: Add missing option value 'single'.  Report from
    Sergei Trofimovich <slyich@gmail.com>.

diff --git a/tp/Texinfo/XS/parsetexi/end_line.c b/tp/Texinfo/XS/parsetexi/end_line.c
index a5bd0ce52a..c973cc1230 100644
--- a/tp/Texinfo/XS/parsetexi/end_line.c
+++ b/tp/Texinfo/XS/parsetexi/end_line.c
@@ -778,8 +778,8 @@ kdbinputstyle_invalid:
     case CM_headings:
       {
         if (!strcmp (line, "off") || !strcmp (line, "on")
-            || !strcmp (line, "double") || !strcmp (line, "singleafter")
-            || !strcmp (line, "doubleafter"))
+            || !strcmp (line, "single") || !strcmp (line, "double")
+            || !strcmp (line, "singleafter") || !strcmp (line, "doubleafter"))
           {
             ADD_ARG(line);
           }
